tests/assets/test_pb2.pyi

```diff
@@ -72,25 +75,40 @@ class MeasurementParameter(google.protobuf.message.Message):
     uint64_data: builtins.int
     bool_data: builtins.bool
     string_data: builtins.str
+    enum_data: builtins.str
```

Comment:
@WesleyTangNationalInstruments  @bkeryan 
 I added these 2 lines here since when I ran the updated version of mypy tests I was getting this error 
tests\unit\test_serializer.py:337: error: "MeasurementParameter" has no attribute "enum_data"  [attr-defined]
tests\unit\test_serializer.py:338: error: "MeasurementParameter" has no attribute "enum_array_data"; maybe "int32_array_data"

Reply:
It was an update that did Brad at the same time you submitted Wes. 
https://github.com/ni/measurementlink-python/pull/292